kinder/pkg/cluster/manager/actions/kubeadm-join.go

```diff
@@ -106,9 +106,10 @@ func kubeadmJoinControlPlane(cp *status.Node, patchesDir, ignorePreflightErrors
 		fmt.Sprintf("--v=%d", vLevel),
 	}
 	if patchesDir != "" {
-		joinArgs = append(joinArgs, "--experimental-patches", constants.PatchesDir)
+		if cp.MustKubeadmVersion().LessThan(constants.V1_21) {
+			joinArgs = append(joinArgs, "--experimental-patches", constants.PatchesDir)
+		}
```

Comment:
i think we can remove this check above:
https://github.com/kubernetes/kubeadm/blob/47c0def8aff991e23c599e207b66edb91632c4fa/kinder/pkg/cluster/manager/actions/kubeadm-join.go#L46-L54

and add copyPatchesToNode here.